Assets/Scripts/Board/Graph.cs

```diff
@@ -80,17 +97,47 @@ public class TileNode : Node
 
     public Graph() { }
 
+    public static List<VertexDirection> GetVertexDirectionNeighbors(VertexDirection vertexDirection)
+    {
+        return vertexDirection switch
+        {
+            VertexDirection.North => new() { VertexDirection.Northwest, VertexDirection.Northeast },
+            VertexDirection.Northeast => new() { VertexDirection.North, VertexDirection.Southeast },
+            VertexDirection.Southeast => new() { VertexDirection.Northeast, VertexDirection.South },
+            VertexDirection.South => new() { VertexDirection.Southeast, VertexDirection.Southwest },
+            VertexDirection.Southwest => new() { VertexDirection.South, VertexDirection.Northwest },
+            VertexDirection.Northwest => new() { VertexDirection.Southwest, VertexDirection.North },
+            _ => new() { },
+        };
+    }
```

Comment:
wut

Reply:
so I get it's a switch that returns the two neighboring vertices to the given vertex on the current tile. but..man I wish we didn't need this. So you use it when adding the new vertices so you know who to get to add an edge between. hmm

Reply:
Resolved Below.